data/bluetooth.appdata.xml.in

```diff
@@ -6,14 +6,14 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>LGPL-2.1</project_license>
   <name>Bluetooth Indicator</name>
-  <summary>Connect and disconnect paired bluetooth devices</summary>
+  <summary>Connect and disconnect paired Bluetooth devices</summary>
   <icon type="stock">preferences-bluetooth</icon>
   <releases>
     <release version="2.1.6" date="2020-12-23" urgency="medium">
       <description>
         <p>Prevent showing duplicate devices</p>
         <p>Fix animated icon when disabling Bluetooth</p>
-        <p>Fallback device names to their device type instead of "(null)"</p>
+        <p>Fallback of device names to their device type instead of "(null)"</p>
```

Comment:
This is less good English imo.  However the original is not good either (fallback is not really a transitive verb). 

I suggest "Device names fallback to their device type instead of '(null)'"

Reply:
```suggestion
        <p>Use device type as fallback instead of "(null)"</p>
```